packages/oui-form-actions/src/index.spec.js

```diff
@@ -0,0 +1,118 @@
+                submit: "Submit",
+                cancel: "Cancel"
+            });
+        });
+
+        beforeEach(inject(_ouiFormActions_ => {
+            ouiFormActions = _ouiFormActions_;
+        }));
+
+        it("should have custom values", () => {
+            expect(ouiFormActions.translations.submit).toEqual("Submit");
+            expect(ouiFormActions.translations.cancel).toEqual("Cancel");
+        });
+    });
+
+    describe("Component", () => {
+        it("should have right button classes", () => {
+            const component = testUtils.compileTemplate(`
+                <oui-form-actions
+                    on-submit="$ctrl.submit()"
+                    on-cancel="$ctrl.cancel()">
+                </oui-form-actions>`);
+            const submitButton = component.find("button").eq(0);
+            const cancelButton = component.find("button").eq(1);
+
+            expect(submitButton.hasClass("oui-button_primary")).toBe(true);
+            expect(cancelButton.hasClass("oui-button_link")).toBe(true);
+        });
+
+        it("should display buttons with default text values", () => {
+            const component = testUtils.compileTemplate(`
+                <oui-form-actions
+                    on-submit="$ctrl.submit()"
+                    on-cancel="$ctrl.cancel()">
+                </oui-form-actions>`);
+            const submitButton = component.find("button").eq(0);
+            const cancelButton = component.find("button").eq(1);
+
+            expect(submitButton.text().trim()).toBe("Submit");
+            expect(cancelButton.text().trim()).toBe("Cancel");
+        });
+
+        it("should display buttons with custom text values", () => {
+            const component = testUtils.compileTemplate(`
+                <oui-form-actions
+                    on-submit="$ctrl.submit()"
+                    on-cancel="$ctrl.cancel()"
+                    submit-text="testSubmit"
+                    cancel-text="testCancel">
+                </oui-form-actions>`);
+            const submitButton = component.find("button").eq(0);
+            const cancelButton = component.find("button").eq(1);
+
+            expect(submitButton.text().trim()).toBe("testSubmit");
+            expect(cancelButton.text().trim()).toBe("testCancel");
+        });
+
+        it("should disable only submit button", () => {
+            const component = testUtils.compileTemplate(`
+                <oui-form-actions
+                    on-submit="$ctrl.submit()"
+                    on-cancel="$ctrl.cancel()"
+                    disabled>
+                </oui-form-actions>`);
+            const submitButton = component.find("button").eq(0);
+            const cancelButton = component.find("button").eq(1);
+
+            expect(submitButton.attr("disabled")).toBe("disabled");
+            expect(cancelButton.attr("disabled")).not.toBe("disabled");
+        });
+
+        it("should trigger click on submit and cancel button", () => {
```

Comment:
It would be better if you separate the test of the submit and cancel. Here, even if you have inverted submit with callback in the controller, the test will pass.

Reply:
Yes !